tango_simlib/utilities/helper_module.py

```diff
@@ -178,7 +178,7 @@ def json_loads_byteified(json_text):
 
 
 def _byteify(data, ignore_dicts=False):
-    """If this is a unicode string, return its string representation."""
+    # If this is a unicode string, return its string representation.
```

Comment:
Why was the docstring changed to a comment?

Reply:
@ajoubertza judging from the following comments i think this was meant tp be a comment more for the if statement that follows rather than a docstring for the entire fuction